INSTALL.md

```diff
@@ -0,0 +1,86 @@
+# Installation Instructions
+
+## Ilios 3 - Frontend vs. Backend
+
+Ilios 3 uses a [Symfony](https://symfony.com/) ([PHP](https://php.net))/SQL backend that provides an API to the Ilios 3 Frontend that is built in javascript using [EmberJs](https://emberjs.com).  The Ilios 3 Frontend is managed by [Ilios development team](https://iliosproject.org) and the latest frontend javascript is served to users via the [Amazon Cloudfront](http://aws.amazon.com/cloudfront/) CDN Content Delivery Network).
+
+## Frontend 
+In most cases, you will never need to deplay the Ilios 3 Frontend on your own servers, as the latest version will always be up-to-date at the CDN.
+
+## Backend
+To build/deploy the Ilios 3 backend, install the default Ilios database and then follow these steps on your webserver(s) to connect to the database and serve the API.
+
+*NOTE:* If you are updating to Ilios 3 from Ilios 2.x for the first time, please backup your database and see Updating Ilios 2.x to Ilios 3 before reading further.  If you have already updated your database schema to the one required for Ilios 3, the following steps will describe how to update the backend code.
+
+These steps assume that you are deploying the Ilios 3 backend via a Git clone from the Ilios repository at https://github.com/ilios/ilios.git and building the install using PHP's 'composer' package manager. For installation from the Ilios 3 source distribution files, please see Installing the Ilios 3 backend from distribution file.
+
+All the steps below should be performed in the context of the user that runs your webserver process (typically, 'apache')
+
+1. Log into your webserver(s) and change to your web server root ('/web/ilios3' for this example).
+2. Clone/update Ilios 3 directory tree to this directory (assuming 'git clone' for this example).
+```bash
+sudo -u apache git clone https://github.com/ilios/ilios.git
+```
+This will create a folder named 'ilios' in your server root directory.  The entire application source tree will be downloaded to this folder and when this process is finished, the 'web' subfolder should be set as your web server's document root ('/web/ilios3/ilios/web') in your webserver configuration.  
+3. Change into the newly created folder
+
+```bash
+cd ilios
+```
+You should now be in the '/web/ilios3/ilios'  
+4. Checkout the correct release version, based on its release tag ('v.3.0.0-beta1' for this example):
+```bash
+sudo -u apache git checkout tags/v3.0.0-beta1
+```   
+5. Run the following command to build the packages and its dependencies.  This step assumes you have PHP 5.4+ and Composer installed on your system:
+```bash
+sudo -u apache composer install
+```  
+This will install the required PHP Symfony packages and their dependencies.  When the process nears completion, you will be prompted with the following configuration setting options.  You should set them as noted:
+```bash
+database_driver: pdo_mysql
```

Comment:
i'm nitpicking here, but IMO it would be better to show the comments above or behind the corresponding config setting, rather than below.